lib/srv/db/sqlserver/kinit/kinit_test.go

```diff
@@ -170,10 +175,11 @@ func TestKRBConfString(t *testing.T) {
 		CommandConfig{
 			User:        "alice",
 			Realm:       "example.com",
-			KDCHost:     "host.example.com",
+			KDCHost:     "instance.host.example.com",
```

Comment:
Can `KDCHost` be empty?

Reply:
No, users should provide it, even if it is the same as their domain.

Reply:
Is `KDCHost` passed in from config as `kdc_host_name` now? For older versions, it would be empty?

Reply:
> Is KDCHost passed in from config as kdc_host_name now? For older versions, it would be empty?


Yes. In older versions, its value would always be empty. [But PKINIT won't be used if there is no KDCHost and LDAPCert.](https://github.com/gravitational/teleport/blob/master/lib/srv/db/sqlserver/connect.go#L66)